src/olympia/versions/models.py

```diff
@@ -493,8 +493,12 @@ def sources_provided(self):
         return bool(self.source)
 
     @property
-    def admin_review(self):
-        return self.addon.admin_review
+    def needs_admin_code_review(self):
+        return self.addon.needs_admin_code_review
+
+    @property
+    def needs_admin_content_review(self):
+        return self.addon.needs_admin_content_review
```

Comment:
Can we drop these two properties?  Seems like they are unnecessary shortcuts.

Reply:
They are necessary to make flags work on the review page itself (`get_flags()` is passed a `Version`)